test/fixtures/autofix/jQueryNativeReplacements.js

```diff
@@ -5,7 +5,7 @@ sap.ui.define(["sap/ui/thirdparty/jquery"], async function (jQuery) {
 	// 1. Replace global "new sap.base.i18n.ResourceBundle()"
 	// 2. Replace the code with the following:
 	// -> var isBundle = new ResourceBundle() instanceof ResourceBundle;
-	// var isBundle = jQuery.sap.resources.isBundle(new sap.base.i18n.ResourceBundle());
+	var isBundle = jQuery.sap.resources.isBundle(new sap.base.i18n.ResourceBundle());
```

Comment:
why is this in a different file than the `jQuery.sap.resources` fixture?

Also, this code does not work at runtime, as `sap.base.i18n.ResourceBundle` is not available as a global (module only).

Reply:
Disabling this test again. It requires double run in this case. Maybe, it will start working when we implement replacement up to the braces as is the plan for `dellayedCall`

Reply:
As the `sap.base.i18n.ResourceBundle` part is not valid (see my comment above), the described problem will not appear. But I'm still fine with not enabling this fix for now. I will just remove the fixture code as it is not valid.

Reply:
This case is the same as https://github.com/SAP/ui5-linter/pull/614#discussion_r2086482801

Basically, the argument of `isBundle()` is wiped out during replacement.
IMO, we can handle thise cases during the `dellayedCall` migration enablement